src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -2,201 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Composition;
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Recommendations;
-using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.Options.Providers;
 
 namespace Microsoft.CodeAnalysis.Completion
 {
     internal record struct CompletionOptions(
```

Comment:
so the type stays at the feature level (As pure data), but hte metadata moves to editor layer, as more policy/storage?

Reply:
Yes.